spec/requests/v1/pacbio/runs_spec.rb

```diff
@@ -15,6 +15,17 @@
       post v1_pacbio_runs_path, params: body, headers: json_api_headers
       expect(response).to have_http_status(:success), response.body
     end
+
+    it 'publishes volume tracking message for each used aliquot' do
+      allow(Emq::Publisher).to receive(:publish)
+
+      post v1_pacbio_runs_path, params: body, headers: json_api_headers
+
+      expect(response).to have_http_status(:success), response.body
+      run = Pacbio::Run.first
+      # Verify that the publish method was called with the expected arguments
+      expect(Emq::Publisher).to have_received(:publish).with(array_including(run.all_library_aliquots), instance_of(Pipelines::Configuration::Item), 'volume_tracking') # rubocop:disable RSpec/MessageSpies
```

Comment:
I think we only need to test that a publish message has been received. I presume the publish message is also tested elsewhere.

Reply:
No. That's fine. As long as the test is generic as possible and the publish message is tested correctly elsewhere.

Reply:
Yes, it checks for aliquots being passed in as part of the publishing_job_spec, but in a generic way — not specifically for runs created with libraries. I don't think this check to see only library aliquots exists  is necessary, as it will be made redundant by pool aliquots in upcoming stories. Additionally, the content of the message is already covered by message_builder_spec